managed/services/agents/log_level.go

```diff
@@ -25,9 +25,15 @@ import (
 // Log level available in exporters with pmm 2.28
 var exporterLogLevelCommandVersion = version.MustParse("2.27.99")
 
-func withLogLevel(args []string, logLevel *string, pmmAgentVersion *version.Parsed) []string {
-	if pointer.GetString(logLevel) != "" && !pmmAgentVersion.Less(exporterLogLevelCommandVersion) {
-		args = append(args, "--log.level="+pointer.GetString(logLevel))
+func withLogLevel(args []string, logLevel *string, pmmAgentVersion *version.Parsed, supportLogLevelFatal bool) []string {
+	level := pointer.GetString(logLevel)
+
+	if level != "" && !pmmAgentVersion.Less(exporterLogLevelCommandVersion) {
+		if !supportLogLevelFatal && level == "fatal" {
+			level = "error"
+		}
```

Comment:
It's need to fix problem for already added exporters with `fatal` level? Can you please add comment with description why it's needed.

Reply:
> It's need to fix problem for already added exporters with `fatal` level?

Yes, it's need to fix problem for already added exporters with `fatal` level

> Can you please add comment with description why it's needed.

Already in variable name `!supportLogLevelFatal`

Reply:
It's not so obvious why we we can face situation where fatal level is not supported AND we got fatal level. And as you confirmed it's possible for previously added exporters.

Reply:
@artemgavrilov i just now add comment